gui/src/data.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+//mock data
+export const scenarios = [
```

Comment:
The data this component will receive will be in the format specified:
[
["cycle id", "cycle label", item type (0 cycle, 1 scenario), false, [
  ["scenario id", "scenario label", item type, primary (boolean value)],
  ...
  ["scenario id", "scenario label", item type, primary (boolean value)]
],
["cycle id", "cycle label", item type (0 cycle, 1 scenario), false, [
  ["scenario id", "scenario label", item type, primary (boolean value)],
  ...
  ["scenario id", "scenario label", item type, primary (boolean value)]
],
...
["scenario id", "scenario label", item type, primary (boolean value)],
  ...
["scenario id", "scenario label", item type, primary (boolean value)]
]